README.md

```diff
@@ -41,9 +41,9 @@ RAG is a method that tries to avoid hallucination by using information that is p
 4. The query generated from the last step is fed to an LLM model, and the **LLM model gives the user an answer based on the query** generated from the last step. The user receives this answer.
 
 <p align="center">
-<img width="1225" alt="Screenshot 2024-10-12 at 02 27 44" src="https://github.com/user-attachments/assets/bf6537c6-d50e-45cc-bb2f-caac0f3d90fe">
+	<img width="1004" alt="Screenshot 2024-12-02 at 03 36 32" src="https://github.com/user-attachments/assets/0a752600-ed20-45eb-a0db-d8539dcb2465">
```

Comment:
`nit` I'm not confident, but "embedded" and "embedding" are completely different things, and using embedded might be misleading...? 🤔

Reply:
is it common to use LLM twice? one is to combine knowledge and user query, and the other is to actually generate an answer

Reply:
I'm glad you are becoming familiar with diagrams.net 👍

Reply:
I will check how embedded and embedding are different and make it clearer....!

I don't know how common it is, but using LLM twice does not seem necessary. I will see how common it is. We are not using it twice in our project right now either. 🤔 🤔 

Thank you for instructing me to use it! That helps my life a lot :)

Reply:
I'll change the figure too if using LLM twice is not common!

Reply:
I don't think using LLM twice is common, I think my understanding when I made the previous figure was incorrect.
I looked up some posts about RAG, and most of them didn't use LLM twice. 
This page explains several kinds of RAG architecture: https://humanloop.com/blog/rag-architectures

I will change the figure👍

Reply:
I don't think embedded and embedding have different meanings, but what do you think?

I feel the usage is like this:
I embed the sequence every day as my day job. 
I embedded the sequence last night.
The sequence is embedded by an expert.
I am embedding the sequence right now. 
Here is an embedding of the sequence.

Reply:
@carolina-museum 
I think you are right, but I have never encountered "embed" being used in the context of generating embeddings though it is grammatically correct to use "embed" for this case (I don't know why 🤔 ) Instead, "embedding" is more commonly used as a noun, for example, when referring to "storing word embeddings."

I also browsed some web pages and searched for "embed(ed)", but I could not find it being used in this way on these pages.
- https://www.cloudflare.com/learning/ai/what-are-embeddings/
- https://en.wikipedia.org/wiki/Word_embedding
- https://en.wikipedia.org/wiki/Embedding
- https://aws.amazon.com/what-is/embeddings-in-machine-learning/
- https://pytorch.org/docs/stable/generated/torch.nn.Embedding.html
- https://learn.microsoft.com/en-us/azure/architecture/ai-ml/guide/rag/rag-generating-embeddings

but maybe I'm saying something stupid. Because I did not come up with helpful resources about this topic, let me just mark this thread as resolved

Reply:
I see, thank you for the extended research!
I see that people try using the noun form most of the time. I added the noun usage too in the description.

For me, it sounds more intuitive to use "embedded" because it implies that the embeddings are made from what is being embedded. But! I will use more "embeddings" next time!